src/main/java/com/google/cloud/resourcemanager/spi/v1beta1/ResourceManagerRpc.java

```diff
@@ -56,6 +60,28 @@ Integer getInt(Map<Option, ?> options) {
     }
   }
 
+  class ListResult<T> {
+
+    private final Iterable<T> results;
+    private final String pageToken;
+
+    public ListResult(String pageToken, Iterable<T> results) {
```

Comment:
constructor is public but class isn't. Same for methods below. Make consistent or add `@Override` as needed

Reply:
done